examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/BananaNapActivity.java

```diff
@@ -13,8 +16,11 @@
  */
 @ActivityType("BananaNap")
 public final class BananaNapActivity {
+  @ActivityType.FixedDuration
```

Comment:
I was wondering whether it would be possible to enable passing the duration into the annotation directly - a quick attempt seems to indicate that it's not possible - so I think the two ways provided in this PR are indeed the best we can do

![Screenshot 2023-04-13 at 6 57 58 AM](https://user-images.githubusercontent.com/1189602/231782293-b065e6dd-da5f-48e0-9042-a31d9005477f.png)